### DISCUSSION TITLE
[Feature] Suggest similar people for face detection

### DISCUSSION THREAD

**Discussion by ASTRELION** [immich-app/immich#14420] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In Google Photos, you can opt to go through suggestions of people that might actually be the same person but were grouped separately. I would like a feature similar to this in Immich since it makes it much easier to group faces together without manually going through the people list.

Under the hood, this could work by Immich lowering the threshold for similar faces in another pass in an attempt to find new pairs. If a pair of people match in this reduced threshold, then suggest to the user that the faces might actually be the same person and allow them to confirm (link the faces together) or deny (keep them as is) the similarity. Optionally, this also could allow a higher default threshold for more accurate faces, while offering an easy way to group more similar faces through an easy-to-use interface.

This would partially alleviate https://github.com/immich-app/immich/discussions/12328.
Somewhat related to https://github.com/immich-app/immich/discussions/4509.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4347
